src/main/java/ua/softserveinc/tc/util/DateUtil.java

```diff
@@ -41,6 +43,10 @@ public static Date toDate(String date) {
         }
     }
 
+    public static Date toDate(LocalDate date) {
```

Comment:
This method is called toDate() but in fact it produces start of the day according to system default ZoneId. So the name of method is misleading. Or you have some other thoughts that I don't understand.